src/main/resources/db/migration/V16__set_default_dataset_updated_at.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE datasets ALTER COLUMN updated_at SET DEFAULT current_timestamp;
```

Comment:
We should only set `updated_at` when the row has been modified. Meaning, a `null` value for `updated_at` in `datasets` signifies the row has been _inserted_, but not yet _updated_. Why change this behavior?